src/commands/base-command.ts

```diff
@@ -103,6 +105,16 @@ async function selectWorkspace(project: Project, filter?: string): Promise<strin
     log()
     log(chalk.cyan(`We've detected multiple sites inside your repository`))
 
+    if (isCI) {
+      throw new Error(
+        `Sites detected: ${(project.workspace?.packages || [])
+          .map((pkg) => pkg.name || pkg.path)
+          .join(
+            ', ',
+          )}. Configure the site you want to work with and try again. See docs at: https://docs.netlify.com/configure-builds/monorepos/#manual-configuration`,
```

Comment:
We typically refrain from using "see" as it's considered ableist language. I've also created a short link to make the message easier on the eye, and lets us point the link to a different part of our docs in the future without updating the CLI.

```suggestion
          )}. Configure the site you want to work with and try again. Refer to https://ntl.fyi/configure-site for more information.`,
```